corehq/apps/api/resources/v0_5.py

```diff
@@ -845,9 +849,44 @@ def obj_get_list(self, bundle, **kwargs):
         if not application:
             return []
         forms_objects = application.get_forms(bare=False)
+
         for form_object in forms_objects:
             form = form_object['form']
             module = form_object['module']
             form_name = '{} > {} > {}'.format(application.name, module.name['en'], form.name['en'])
             results.append(Form(form_xmlns=form.xmlns, form_name=form_name))
         return results
+
+#Zapier requires id and name; case_type has no obvious id, placeholder inserted instead.s
```

Comment:
nit: dangling `s`. Also, in [pep8](https://www.python.org/dev/peps/pep-0008/#inline-comments) comments have a space after the `#`